### ISSUE TITLE
Assumptions of bash in BUILDING.md

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#2043]
The install commands for prerequisites for RHEL etc are using bash syntax like

`sudo yum localinstall --nogpgcheck https://download1.rpmfusion.org/free/el/rpmfusion-free-release-$(rpm -E %rhel).noarch.rpm
`

However, if you are using cshell (like my colleague) then the subshell command `$(rpm -E %rhel)` simply prints "no such variable" or similar.

Maybe need to warn people that the syntax below uses bash sub-shell.  If on cshell you must run `rpm -E %rhel` manually and substitute the result (e.g., 7) into the command, e.g.,

`sudo yum localinstall --nogpgcheck https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm

**Comment by PaulWessel** [comment]
Hi @seisman, is there an equivalent csh sub-shell thingy to use?

**Comment by seisman** [comment]
This works for both bash and csh.
```
https://download1.rpmfusion.org/free/el/rpmfusion-free-release-`rpm -E %rhel`.noarch.rpm
```

**Comment by seisman** [comment]
Closed by #2062.
